### DISCUSSION TITLE
Componentes instaláveis (Delphi)

### DISCUSSION THREAD

**Discussion by Antonio-Carllos** [HashLoad/boss#102] [Q&A] [answered]
É possível adicionar uma dependência de uma componente instalável??

**Comment by viniciussanchez** [comment, marked as answer]
As dependências são de projetos que o boss vai instalar automaticamente quando você instalar a dependência em questão.
Ex: 
projeto A  depende do projeto B e C
ao instalar o projeto A, o boss vai instalar o B e o C

Se o boss nao consegue instalar o B e o C, nao teria muita finalidade colocar a dep.

**Reply by Antonio-Carllos** [reply]
Assim, entendo. Uma outra duvida, o boss consegue instalar componentes?

**Reply by viniciussanchez** [reply]
Sim, mas nao todos.

**Reply by Antonio-Carllos** [reply]
Os componentes que eu gero no padrão de .dpk e instalo  no Delphi por exemplo, ele conseguiria instalar?

**Reply by viniciussanchez** [reply]
Sim, consegue sim...

**Reply by viniciussanchez** [reply]
precisa ter instalado o boss-ide para isso funcionar

**Reply by Antonio-Carllos** [reply]
Eu testei na versão do Delphi Alexandria, mas não conseguir fazer funcionar, Existe alguma documentação dos passos?

**Reply by viniciussanchez** [reply]
Acho que o @gabrielbaltazar tem um vídeo sobre... mas o que de fato não funcionou? Instalou o boss ide?

**Reply by Antonio-Carllos** [reply]
Instalei sim, tentei de duas formas.. a primeira fiz pelo comando "boss install -g boss-ide", a outra fiz de forma manual, baixando a release a installando...

**Reply by Antonio-Carllos** [reply]
O Componente que fiz para teste não foi carregado na paleta de ferramenta de um outro projeto demo.

**Reply by viniciussanchez** [reply]
O seu projeto tem os arquivos do boss?

veja se está parecido com isso:

![image](https://user-images.githubusercontent.com/16382981/173842750-3a4f36ea-3ebf-46c0-8ff9-fb6191bebee5.png)

**Reply by Antonio-Carllos** [reply]
O projeto tem o arquivo do boss, só o atributo "projects" estava sem informação.

**Reply by Antonio-Carllos** [reply]
Colocando o .dproj está gerando um erro:
![Sem título](https://user-images.githubusercontent.com/67807222/174318881-f38515fa-0037-4f38-9afc-b6b666a6df15.png)
O arquivos está na seguinte forma:
![Sem título 1](https://user-images.githubusercontent.com/67807222/174319285-3e95c704-f5f0-4744-9175-fceea770df6b.png)

**Reply by arvanus** [reply]
Acho que falou as aspas aí

**Reply by Antonio-Carllos** [reply]
Realmente esse era o problema, obrigado galera!
